src/cljs/harja/views/tilannekuva/tilannekuva.cljs

```diff
@@ -151,17 +149,15 @@
                          (conj ryhma-polku elementti)]))])]))))))
 
 (defn- asetuskokoelma
-  [otsikko {:keys [salli-piilotus? luokka auki-atomi? otsikon-luokka] :as optiot} sisalto]
+  [otsikko {:keys [ei-nappi? salli-piilotus? luokka auki-atomi? otsikon-luokka] :as optiot} sisalto]
```

Comment:
Negaatio-asetus vaikuttaa vähän hankalalta.
Oisko parempi olla jokin default asetus nappi? true, jos se on yleisempää
Ja jos ei halua nappia, niin nappi? false

Reply:
On vain yksi paikka, missä ei-nappi? olisi true, sillä ajattelin että näin default menee helpommin aina oikein.

Reply:
Ymmärrän.
Yleensä käytäntönä on käyttää positiivisia nimia asetuksille, niin ei tule sitten esimerkiksi hämäävää tuplanegatiivia, jos jossain pitää laittaa vaikka (not ei-nappi?) tms. 
Niin, ajattelin huomauttaa vaikka ei iso juttu juuri tässä olekaan :)

Reply:
Kyllä tämä oli paikallaan oleva hyvä kommentti. Voin tietenkin käyttää eksplisiittistä false? tsekkiä tähän, eli tällöin antamaton arvo, eli defualt, on nappi. Kiitos.